rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -317,6 +322,9 @@ private String findDatedSnapshotVersionIfNecessary(String groupId, String artifa
 
     @Nullable
     private MavenRepository normalizeRepository(MavenRepository originalRepository) {
+        if (!normalizeRepositories) {
```

Comment:
You may ask yourself, why add an ability to disable this repository testing. The root repository url throws a 404 for gcp artifact registry maven repositories, so this was failing.